platform/linuxbsd/wayland/dynwrappers/wayland-egl-core.c

```diff
@@ -0,0 +1,67 @@
+// This file is generated. Do not edit!
+// see https://github.com/hpvb/dynload-wrapper for details
+// generated by ./generate-wrapper.py 0.3 on 2022-10-24 14:46:56
+// flags: ./generate-wrapper.py --include /usr/include/wayland-egl-core.h --sys-include <wayland-egl-core.h> --soname libwayland-egl.so --init-name wayland_egl --output-header wayland-egl-core.h --output-implementation wayland-egl-core.c
```

Comment:
And here, https://packages.debian.org/sid/amd64/libwayland-egl1/filelist (`libwayland-egl.so.1`)